src/main/kotlin/no/nav/arbeidsgiver/min_side/models/AltinnTilgangssøknad.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.arbeidsgiver.min_side.models
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class AltinnTilgangssøknad(
+    var orgnr: String? = null,
```

Comment:
null vs "" ? gikk for null på noen, mens "" på der det ble til mange dobbel bangs